CHANGELOG.md

```diff
@@ -53,3 +53,16 @@ Additions to existing modules
   ```agda
   _≡?_ : DecidableEquality (Vec A n)
   ```
+
+* In `Relation.Nullary.Decidable`:
+  ```agda
+  does-⇔  : A ⇔ B → (a? : Dec A) → (b? : Dec B) → does a? ≡ does b?
+  does-≡  : (a? b? : Dec A) → does a? ≡ does b?
+  ```
+
+* In `Relation.Nullary.Properties`:
```

Comment:
We missed this (copy/paste error?) during review: should be `Relation.Unary.Properties`!

Reply:
Nice spot! Any chance of a quick PR?

Reply:
Well, I have already fixed it on (a couple of ;-)) open branches as part of merge conflict resolution, so the alternative would be to merge those... but if you want a cleaner fine-grained git history, happy to do so.